styleguide/source/assets/js/nav.js

```diff
@@ -10,29 +10,32 @@
  (function ($, Drupal) {
    Drupal.behaviors.nav = {
      attach: function (context, settings) {
-
+        var menuTrigger = '#menu-trigger',
```

Comment:
@aCyborg you should store the jQuery object itself here and not just the selector:
``` var menuTrigger = $('#menu-trigger');```
So that further down you just need to use
``` menuTrigger.click(function() {```